translations/de/info/task_short_description.html

```diff
@@ -0,0 +1 @@
+Mit Zahlen auf nicht gewöhnliche Weise arbeiten.
```

Comment:
A correct translation, of course.

But what do you think about short descriptions beeing a bit more "catchy"...

something like: _Eine etwas andere Art mit Zahlen zu arbeiten._ ???